buffer/outputs.tf

```diff
@@ -59,6 +59,8 @@ output "cloudsql_app_stairway_db_creds" {
 # Pool Folder Outputs
 #
 output "pool_name_to_folder_id" {
+  // The display name on the folder is the pool id. Because the folders are not generated in a specific order
+  // we cannot rely on their order matching the order of the input variable pool_list
```

Comment:
Why can't we zip the `var.pool_list` with `google_folder.pool_folders` to match pool name to folder id? I would really think the ordering of those would be guaranteed to match.

Reply:
oh, because we use `toset` maybe let's not do that.